source/community.applications/usr/local/emhttp/plugins/community.applications/include/force_update_helpers.php

```
@@ -34,11 +34,12 @@ public static function resetTemplatesCache(bool $ensureTemplatesDirectory = fals
 	public static function fetchLatestUpdateMetadata(): array {
 		@unlink(CA_PATHS['lastUpdated']);
 
-		// Ensure force_update cannot hang forever if the remote is stalled.
-		$latestUpdate = download_json(CA_PATHS['application-feed-last-updated'], CA_PATHS['lastUpdated'], 60);
+		/* No cURL timeout — ca_downloadProgress nchan stream surfaces stalls
+		   to the user (with an Abort button). */
+		$latestUpdate = download_json(CA_PATHS['application-feed-last-updated'], CA_PATHS['lastUpdated'], 0);
```

PHPStan (2.1.54)

[error] Constant CA_PATHS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)